scipy/stats/tests/test_distributions.py

```diff
@@ -967,6 +970,89 @@ def test_stats(self):
         assert_allclose(result, expected)
 
 
+class TestKSTwo(object):
+    def setup_method(self):
+        np.random.seed(1234)
+
+    def test_cdf(self):
+        for n in [1, 2, 3, 10, 100, 1000]:
```

Comment:
I suppose you may get more informative test failures with pytest parametrization here and below.